.github/workflows/release.yml

```diff
@@ -209,6 +209,9 @@ jobs:
           DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE: ${{ steps.setup-dct.outputs.dct-delegate-repo-passphrase }}
           SERVICE_NAME: ${{ steps.setup.outputs.service_name }}
         run: |
+          if [ "$SERVICE_NAME" == "Scim" ]; then
```

Comment:
```suggestion
          if [ "$SERVICE_NAME" == "scim" ]; then
```

Reply:
Did you check the Service name?

Reply:
I can see it is changed to lowercase later